.github/workflows/test-psalm.yml

```diff
@@ -24,7 +24,7 @@ jobs:
   build:
     name: Psalm Analysis
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, '[ci skip]')"
+    if: '! contains(github.event.head_commit.message, ''[ci skip]'')'
```

Comment:
PhpStorm reports an error. Is this false positive?
![Screenshot 2024-07-23 13 26 36](https://github.com/user-attachments/assets/e198391e-a9c8-4265-b1d6-d19cfbc941f0)

Reply:
Hmm, it seems we cannot satisfy all worlds.

Could you try this change:
```diff
diff --git a/.github/workflows/test-psalm.yml b/.github/workflows/test-psalm.yml
index b376416e27..145d984b83 100644
--- a/.github/workflows/test-psalm.yml
+++ b/.github/workflows/test-psalm.yml
@@ -24,7 +24,7 @@ jobs:
   build:
     name: Psalm Analysis
     runs-on: ubuntu-latest
-    if: '! contains(github.event.head_commit.message, ''[ci skip]'')'
+    if: (! contains(github.event.head_commit.message, '[ci skip]'))
 
     steps:
       - name: Checkout
```

Reply:
PhpStorm does not complain about it.

Reply:
Resorted to using parentheses instead.